report/sections/abstract.tex

```diff
@@ -0,0 +1,5 @@
+\begin{abstract}
```

Comment:
Should the Abstract section be on its own page? I can't remember if IEEE demands it.